### PR TITLE
add Route

### PR DESCRIPTION
**Pull request by jmohan57**

### CODE REVIEW

**Review by jmohan57 — COMMENTED** [review]
hey, your commit comment is weak.

File: components/SettingsCheckboxes.tsx
Comment (jmohan57) [inline components/SettingsCheckboxes.tsx:1]: leave your line so on.

Comment (jmohan57) [reply components/SettingsCheckboxes.tsx:1]: dfsdf

**Review by venomsnake0777 — COMMENTED** [review]
yes

**Review by jmohan57 — COMMENTED** [review]
got it

**Pull request merged**